tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -1496,6 +1496,15 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
     }
     return false;
   }
+  // For NKL-DNN only int32 supported for axis data type
+  static bool ConcatV2Rewrite(const Node *n) {
+    DataType T;
+    GetNodeAttr(n->def(),"Tidx", &T);
+    if (T != DT_INT32) {
+      return false;
+    }
+    return true;
```

Comment:
Nit: We can simplify this to:
```suggestion
    return (T == DT_INT32);
```